brands/public_transport/station.json

```diff
@@ -0,0 +1,16 @@
+{
+  "public_transport/station|Greyhound Station": {
+    "countryCodes": ["us"],
```

Comment:
Luckily, Greyhound happens to be a national carrier. But as we continue to add public transportation providers, we’re going to run into lots of conflicts across states. Unlike with brands and even banks, most public transportation operators are confined to a single state and their state coverage is quite stable. So we’ll need a way to track [ISO 3166](https://en.wikipedia.org/wiki/ISO_3166) subnational codes, not just country codes, even if we don’t happen to use it for Greyhound.

Reply:
Agreed. We don't want Los Angeles bus stops to be recommended in Boston.